python_modules/dagster/dagster/_config/structured_config/__init__.py

```diff
@@ -143,6 +143,11 @@ def _as_config_dict(self) -> Mapping[str, Any]:
                 output[key] = value
         return output
 
+    @classmethod
+    def to_config_field(cls) -> Field:
+        """Converts the config structure represented by this class into a Dagster config field."""
+        return infer_schema_from_config_class(cls)
+
```

Comment:
how about `to_config_schema` and have it return `DefinitionConfigSchema`? That would prevent possible confusion with pydantic.Field

Reply:
that would also line up with `config_schema` kwarg on the `@resource` et al